### PR TITLE
New tutorials and some minor documentation update

### PR DESCRIPTION
**Pull request by rodrigosiqueira**

### CODE REVIEW

File: documentation/tutorials/find-maintainers.rst
Comment (magalilemes) [inline documentation/tutorials/find-maintainers.rst:14]: Maybe too many colons?

File: documentation/tutorials/find-maintainers.rst
Comment (magalilemes) [inline documentation/tutorials/find-maintainers.rst:16]: Should be `kw maintainers <PATH>`

File: documentation/index.rst
Comment (magalilemes) [inline documentation/index.rst:15]: I think it would be better if it were either "we also have a blog which..." or "we also have blog posts which..."

File: documentation/content/codingstyle.rst
Comment (kwy95) [reply documentation/content/codingstyle.rst:80]: Since you are already here it might be worth updating the placeholders used in the examples to the correct format of placing the text between angle brackets and using lower case letters. So `EXPRESSION` should be `<expression>` and so on.

File: documentation/content/codingstyle.rst
Comment (kwy95) [inline documentation/content/codingstyle.rst:176]: Double back ticks.

File: documentation/content/codingstyle.rst
Comment (kwy95) [inline documentation/content/codingstyle.rst:179]: Double back ticks.

File: documentation/content/codingstyle.rst
Comment (kwy95) [inline documentation/content/codingstyle.rst:187]: Double back ticks.

File: documentation/content/codingstyle.rst
Comment (kwy95) [inline documentation/content/codingstyle.rst:198]: Double back ticks for command arguments and functions, keep the single back ticks for file names.

File: documentation/content/codingstyle.rst
Comment (kwy95) [inline documentation/content/codingstyle.rst:207]:
```suggestion
    test_target_function_name[_<description>]()
```

File: documentation/content/codingstyle.rst
Comment (kwy95) [inline documentation/content/codingstyle.rst:241]:
```suggestion
executed. This way the origin of an error message can quickly be identified by
```

File: documentation/content/codingstyle.rst
Comment (kwy95) [inline documentation/content/codingstyle.rst:252]:
```suggestion
using the ``build`` option in the `build.sh`, and for ``configm`` in the file
```

**Review by kwy95 — CHANGES_REQUESTED** [review]
I added some notes, also, as per our 'how to contribute' documentation the commit messages titles should be lower case and the message should include a body whenever possible.

**Comment by rodrigosiqueira** [comment]
Hi @magalilemes @kwy95 ,
I addressed all your comments, and after that, I merged this PR with the unstable branch.
Thanks a lot for your review.

**Pull request closed without merging**
